### DISCUSSION TITLE
How to allow upgrade of Akka?

### DISCUSSION THREAD

**Discussion by honzabazant** [scala-steward-org/scala-steward#3172] [Q&A] [answered]
Hi everyone,
our organization recently obtained licence from Lightbend that allows us to use Akka with BSL licence.
Because its update is disabled by default, I would like to ask what should I do to enable it?
I looked into the documentation, but I did not find a way to override such global settings.
Thanks in advance.

**Comment by sebastian-alfers** [comment, marked as answer]
It seems you can provide your own `scala-steward.conf` file as mentioned [here](https://github.com/scala-steward-org/scala-steward/issues/2708#issuecomment-1488516199).
